cms/templates/js/certificate-editor.underscore

```diff
@@ -0,0 +1,45 @@
+<form class="collection-edit-form certificate-edit-form">
+    <% if (error && error.message) { %>
+        <div class="certificate-edit-error message message-status message-status error is-shown" name="certificate-edit-error">
+          <%= gettext(error.message) %>
+        </div>
+    <% } %>
+    <div class="wrapper-form">
+        <fieldset class="collection-fields certificate-fields">
+            <legend class="sr"><%= gettext("Certificate Information") %></legend>
+            <div class="input-wrap field text required add-collection-name add-certificate-name <% if(error && error.attributes && error.attributes.name) { print('error'); } %>">
+                <label for="certificate-name-<%= uniqueId %>"><%= gettext("Certificate Name") %></label>
+                <% if (!_.isUndefined(id)) { %>
+                    <span class="certificate-id">
+                        <span class="certificate-label"><%= gettext('Certificate ID') %></span>
+                        <span class="certificate-value"><%= id %></span>
+                    </span>
+                <% } %>
+                <input id="certificate-name-<%= uniqueId %>" class="collection-name-input input-text" name="certificate-name" type="text" placeholder="<%= gettext("This is the Name of the Certificate") %>" value="<%= name %>">
```

Comment:
I'm not seeing this placeholder text in the sandbox. I'm seeing "Default Name" instead.
The default text in fields on this page seems to be darker than what we use on other Studio pages, e.g. Group Configurations or Libraries.